variables.tf

```diff
@@ -383,7 +383,7 @@ variable "helm_wait_for_jobs" {
 
 variable "helm_skip_crds" {
   type        = bool
-  default     = false
+  default     = true
```

Comment:
1. Incorporated this functionality using `helm_skip_crds` variable. When it is set to `true` no CRDs will be installed.
2. This is tricky. We want to have CRDs upgradability, but we need to rely on Karpenter maintainers to keep previous CRDs and versions in place. So far they did with 0.32 release. But with 0.33 old CRDs are gone. With installation via ArgoCD we are find as ArgoCD won't prune old resources by default. Only Helm installation is problematic. But it would be still viable to do a upgrade "version by version", i.e. "0.29 -> 0.32 (migrate CRDs) -> 0.33"